setup.cfg

```diff
@@ -61,47 +109,41 @@ xfail_strict = True
 filterwarnings =
     error:Sparse:FutureWarning
     error:The SparseArray:FutureWarning
-junit_family=xunit2
+junit_family = xunit2
 
 [codespell]
-ignore-words-list=ba,blocs,coo,datas,fo,hist,nd,ser
+ignore-words-list = ba,blocs,coo,datas,fo,hist,nd,ser
 
 [coverage:run]
 branch = False
 omit =
-     */tests/*
-     pandas/_typing.py
-     pandas/_version.py
+    */tests/*
+    pandas/_typing.py
+    pandas/_version.py
 plugins = Cython.Coverage
 
 [coverage:report]
 ignore_errors = False
 show_missing = True
 omit =
     pandas/_version.py
-# Regexes for lines to exclude from consideration
 exclude_lines =
-    # Have to re-enable the standard pragma
     pragma: no cover
 
-    # Don't complain about missing debug-only code:
     def __repr__
     if self\.debug
 
-    # Don't complain if tests don't hit defensive assertion code:
     raise AssertionError
     raise NotImplementedError
     AbstractMethodError
 
-    # Don't complain if non-runnable code isn't run:
     if 0:
     if __name__ == .__main__.:
     if TYPE_CHECKING:
```

Comment:
Looks like we need to remove empty lines here for the linters to agree
```
exclude_lines =
    pragma: no cover
    def __repr__
    if self\.debug
    raise AssertionError
    raise NotImplementedError
    AbstractMethodError
    if 0:
    if __name__ == .__main__.:
    if TYPE_CHECKING:
```
setup-cfg-fmt` wants consistent indents, `remove-trailing-whitespace` doesn't want trailing whitespaces